### PR TITLE
Added a quickstart for Anaconda

### PR DESCRIPTION
**Pull request by Xaenalt**
Looking for feedback on this format for an Anaconda CE quickstart

### CODE REVIEW

**Comment by taneem-ibrahim** [comment]
@jeff-phillips-18 /PTAL

**Comment by Xaenalt** [comment]
Language likely needs a tiny massage to be ready

File: data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    Red Hat® OpenShift® Data Scientist lets you run Jupyter notebooks on our Red Hat® OpenShift Dedicated environment.
+
+    We have configured the JupyterHub spawner to support your amazing Data Scientist explorations and model deployments using Anaconda.
+
+    We just know you will absolutely love this amazing environment.  This quick start will get you working in a notebook in just a few minutes.
+  tasks:
+    - title: Launch JupyterHub
+      description: |-
+        ### To find the JupyterHub Launch action:
+        1. In the **Applications** menu item, go to **Installed** projects.
```
Comment (jeff-phillips-18) [inline data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml:20]:
```suggestion
        1. In the **Applications** menu item, go to **Enabled** projects.
```

File: data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        2. Find the JupyterHub entry.
+        3. Click **Launch** on the JupyterHub card to access the JupyterHub Spawner.
+
+        A new browser tab window will open displaying the JupyterHub Spawner.
```
Comment (jeff-phillips-18) [inline data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml:24]:
```suggestion
        A new browser tab will open displaying the JupyterHub Spawner.
```

File: data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        failedTaskHelp:
+          This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have a running jupyter notebook
+        failed: Try the steps again.
+    - title: Ensure you are subscribed to use Anaconda Commercial Edition packages.
+      description: |-
+        ### To verify that you are subscribed to Anaconda Commercial Edition in the notebook, follow these steps:
+        1. Create a notebook or open a terminal
+        2. Run conda info (in a notebook, to run terminal commands, start the line with !. For example: !conda info)
```
Comment (jeff-phillips-18) [inline data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml:71]:
```suggestion
        2. Run `conda info` (in a notebook, to run terminal commands, start the line with !. For example: `!conda info`)
```

File: data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        failedTaskHelp:
+          This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have successfully subscribed to Anaconda Commercial Edition
+        failed: Try the steps again.
+    - title: Install TensorFlow as an example of how to install Anaconda Commercial Edition packages
+      description: |-
+        ### To ensure you can install packages, install the Anaconda Commercial Edition TensorFlow package
+        1. Open a notebook or terminal
+        2. Run conda install tensorflow (notebooks will need to prefix this command with !, so !conda install tensorflow)
```
Comment (jeff-phillips-18) [inline data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml:87]:
```suggestion
        2. Run `conda install tensorflow` (notebooks will need to prefix this command with !, so `!conda install tensorflow`)
```

File: data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+      review:
+        instructions: |-
+          #### Verify that you can import TensorFlow
+          1. Inside a notebook, run: import tensorflow as tf
```
Comment (jeff-phillips-18) [inline data/quickstarts/anaconda-ce-create-jupyter-notebook-quickstart.yaml:91]:
```suggestion
          1. Inside a notebook, run: `import tensorflow as tf`
```

**Comment by Xaenalt** [comment]
Gg github >.> why is there not a button to just stage them all or squash all those commits xD

**Review by jeff-phillips-18 — APPROVED** [review]

**Pull request merged**